doc/concepts.qbk

```diff
@@ -199,6 +199,76 @@ the generator, for example to re-run a test suite at a later time.]
 
 [endsect]
 
+[section Quasi-Random Number Generator]
+
+A quasi-random number generator is a __UniformRandomNumberGenerator which
+provides a deterministic sequence of quasi-random numbers, based on some
+algorithm and internal state. [classref boost::random::niederreiter_base2_engine
+Niederreiter base 2] generator is an example of such a [qrng quasi-random
+number generator]. The "quasi" modifier is used to denote more clearly that the
+values produced by such a generator are neither random nor pseudo-random, but
+they form a low discrepancy sequence. The intuitive idea is that a low discrepancy
```

Comment:
The "quasi" modifier is used to indicate that the stream of vectors delivered by the generator form a low-discrepancy sequence.

Reply:
If we use "values" above, then this wording should be kept as it is. There is nothing wrong with it.